components/ide-service/pkg/server/server.go

```diff
@@ -351,6 +354,23 @@ func (s *IDEServiceServer) ResolveWorkspaceConfig(ctx context.Context, req *api.
 			resp.IdeImageLayers = append(resp.IdeImageLayers, desktopImageLayer)
 			resp.IdeImageLayers = append(resp.IdeImageLayers, desktopUserImageLayers...)
 		}
+
+		// we are retunring the actually used ide name here, which might be different from the user's choice
```

Comment:
@mustard-mh Could you take a look? I have extended this service to return the actual ide Settings again, so we can store what was used in the workspace_instance and use it for the next workspace start.

Reply:
maybe add some tests

Reply:
I need to update the golden files